snapcraft/main.py

```diff
@@ -19,82 +19,87 @@
 snapcraft
 
 Usage:
-  snapcraft [--version | --help] [options] COMMAND [ARGS ...]
   snapcraft [options]
+  snapcraft [options] init
+  snapcraft [options] pull [<part> ...]
+  snapcraft [options] build [<part> ...]
+  snapcraft [options] stage [<part> ...]
+  snapcraft [options] strip [<part> ...]
+  snapcraft [options] clean [<part> ...] [--step <step>]
+  snapcraft [options] snap [<directory> --output <snap-file>]
```

Comment:
Does this syntax mean that `--output` is required when specifying `<directory>`, and not possible to specify when _not_ specifying `<directory>`? Should this instead be `snapcraft [options] snap [<directory>] [--output <snap-file>]` ?